since

```
server/src/test/java/org/apache/ambari/server/audit/request/DefaultEventCreatorTest.java:[253,7] method does not override or implement a method from a supertype
[ERROR] /root/code/ambari/ambari-server/src/test/java/org/apache/ambari/server/audit/request/DefaultEventCreatorTest.java:[411,7] method does not override or implement a method from a supertype
```

why don just remove ```@override```?

--- a/ambari-admin/pom.xml
+++ b/ambari-admin/pom.xml
@@ -23,7 +23,6 @@
     <relativePath>../ambari-project</relativePath>
   </parent>
   <modelVersion>4.0.0</modelVersion>
-  <groupId>org.apache.ambari</groupId>
   <artifactId>ambari-admin</artifactId>
   <packaging>jar</packaging>
   <name>Ambari Admin View</name>

--- a/ambari-agent/pom.xml
+++ b/ambari-agent/pom.xml
@@ -23,7 +23,6 @@
     <relativePath>../ambari-project</relativePath>
   </parent>
   <modelVersion>4.0.0</modelVersion>
-  <groupId>org.apache.ambari</groupId>
   <artifactId>ambari-agent</artifactId>
   <name>Ambari Agent</name>
   <description>Ambari Agent</description>

--- a/ambari-funtest/pom.xml
+++ b/ambari-funtest/pom.xml
@@ -16,7 +16,6 @@
     <version>${revision}</version>
     <relativePath>../ambari-project</relativePath>
   </parent>
-  <groupId>org.apache.ambari</groupId>
   <artifactId>ambari-funtest</artifactId>
   <packaging>jar</packaging>
   <name>Ambari Functional Tests</name>

--- a/ambari-project/pom.xml
+++ b/ambari-project/pom.xml
@@ -20,7 +20,6 @@
     <artifactId>ambari</artifactId>
     <version>${revision}</version>
   </parent>
-  <groupId>org.apache.ambari</groupId>
   <artifactId>ambari-project</artifactId>
   <description>Apache Ambari Project POM</description>
   <name>Apache Ambari Project POM</name>

--- a/ambari-server-spi/pom.xml
+++ b/ambari-server-spi/pom.xml
@@ -21,7 +21,6 @@
     <version>${revision}</version>
     <relativePath>../ambari-project</relativePath>
   </parent>
-  
   <artifactId>ambari-server-spi</artifactId>
   <url>http://ambari.apache.org</url>
   <name>Ambari Server SPI</name>

--- a/ambari-server/pom.xml
+++ b/ambari-server/pom.xml
@@ -16,7 +16,6 @@
     <relativePath>../ambari-project</relativePath>
   </parent>
   <modelVersion>4.0.0</modelVersion>
-  <groupId>org.apache.ambari</groupId>
   <artifactId>ambari-server</artifactId>
   <packaging>jar</packaging>
   <name>Ambari Server</name>
@@ -1479,11 +1478,6 @@
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
       <version>${fasterxml.jackson.databind.version}</version>
-    </dependency>
-    <dependency>
-      <groupId>com.fasterxml.jackson.core</groupId>
-      <artifactId>jackson-core</artifactId>
-      <version>${fasterxml.jackson.version}</version>
     </dependency>
     <dependency>
       <groupId>com.fasterxml.jackson.jaxrs</groupId>

--- a/ambari-serviceadvisor/pom.xml
+++ b/ambari-serviceadvisor/pom.xml
@@ -30,8 +30,6 @@
     <version>${revision}</version>
     <relativePath>../ambari-project</relativePath>
   </parent>
-
-  <groupId>org.apache.ambari</groupId>
   <artifactId>ambari-serviceadvisor</artifactId>
   <name>Ambari Service Advisor</name>
   <description>Service Advisor</description>

--- a/ambari-utility/pom.xml
+++ b/ambari-utility/pom.xml
@@ -26,9 +26,7 @@
     <version>${revision}</version>
     <relativePath>../ambari-project</relativePath>
   </parent>
-
   <artifactId>ambari-utility</artifactId>
-  <groupId>org.apache.ambari</groupId>
 
   <dependencies>
     <dependency>
@@ -148,12 +146,6 @@
           <artifactId>commons-beanutils</artifactId>
         </exclusion>
       </exclusions>
-    </dependency>
-    <dependency>
-      <groupId>jakarta.ws.rs</groupId>
-      <artifactId>jakarta.ws.rs-api</artifactId>
-      <version>3.1.0</version>
-      <scope>compile</scope>
     </dependency>
   </dependencies>
 

--- a/ambari-views/pom.xml
+++ b/ambari-views/pom.xml
@@ -24,7 +24,6 @@
         <relativePath>../ambari-project</relativePath>
     </parent>
     <modelVersion>4.0.0</modelVersion>
-    <groupId>org.apache.ambari</groupId>
     <artifactId>ambari-views</artifactId>
     <packaging>jar</packaging>
     <name>Ambari Views</name>

